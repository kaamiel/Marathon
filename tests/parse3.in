addUser 0 01
addMovie 01 01
addMovie 1 02
marathon 01 1
marathon 1 02
delMovie 01 01
delMovie 1 02
delUser 01
