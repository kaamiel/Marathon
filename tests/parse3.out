OK
OK
OK
2
2 1
OK
OK
OK
